summary.sh

```diff
@@ -4,19 +4,19 @@
 # Use its output to guide your analysis of the results of running ./run-dljc.sh: you must manually
 # examine the results of any project that appears in the "unaccounted for" list.
 
-target=$1
+targetdir=$1
 
-total=$(find "${target}" -name "*.log" | wc -l)
+total=$(find "${targetdir}" -name "*.log" | wc -l)
 
-timed_out=$(grep -cl "dljc timed out for" "${target}/*.log")
+timed_out=$(grep -cl "dljc timed out for" "${targetdir}/*.log")
 timed_out_percent=$(((timed_out*100)/total))
-timed_out_list=$(grep -l "dljc timed out for" "${target}/*.log")
+timed_out_list=$(grep -l "dljc timed out for" "${targetdir}/*.log")
 
-no_build=$(grep -cl "no build file found for" "${target}/*.log")
+no_build=$(grep -cl "no build file found for" "${targetdir}/*.log")
 no_build_percent=$(((no_build*100)/total))
 
-no_cf_old=$(grep -cl "dljc could not run the Checker Framework" "${target}/*.log")
-no_cf_new=$(grep -cl "dljc could not run the build successfully" "${target}/*.log")
+no_cf_old=$(grep -cl "dljc could not run the Checker Framework" "${targetdir}/*.log")
+no_cf_new=$(grep -cl "dljc could not run the build successfully" "${targetdir}/*.log")
```

Comment:
"_new" is a funny name.  How about "unbuildable" or "build_failed"?

Reply:
`no_cf` is the semantic name here. At some point during the development of this script, I changed the error message for the same error from "dljc could not run the Checker Framework" to "dljc could not run the build successfully". For backwards compatibility with old experiments, I kept checking for both. New experiments will only find the latter message, but I see no reason to remove it from the code. I'll add a comment to clarify.